src/layouts/Root/components/StyledNavigationLink.tsx

```diff
@@ -5,10 +5,35 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+import React from 'react';
 import styled from 'styled-components';
-import { Link } from 'gatsby';
+import { Link as GatsbyLink } from 'gatsby';
 import { getColor } from '@zendeskgarden/react-theming';
 
+interface ILink {
+  children: string;
+  to: string;
+  activeClassName: string;
+}
+
+const Link = ({ children, to, ...props }: ILink) => {
+  const internal = /^\/(?!\/)/u.test(to);
```

Comment:
There are some alternative strategies for determining relative vs absolute URLS. You may want to take a look at the logic in https://stackoverflow.com/questions/10687099/how-to-test-if-a-url-string-is-absolute-or-relative

Reply:
Oh that's cool. Would we prefer that regex pattern?